Sources/Components/Components/Watchlist/WKWatchlistViewController.swift

```diff
@@ -37,6 +53,17 @@ public final class WKWatchlistViewController: WKCanvasViewController {
 		addComponent(hostingViewController, pinToEdges: true)
 		self.title = viewModel.localizedStrings.title
 		navigationItem.rightBarButtonItem = filterBarButton
+        viewModel.$activeFilterCount.sink { [weak self] newCount in
```

Comment:
Something odd I've been running into: the selected `WKFormSelectSingleRowView` filters aren't always reflecting their count back into the active filter count. Here's an example of selecting "Not latest revision" and the count not being updated in the watchlist view.

![filter_count](https://github.com/wikimedia/wikipedia-ios-components/assets/5652327/49772b97-b831-4108-ad32-7641346a41cb)

Reply:
@staykids Good catch. I have some confusion with interpreting the Android calls here. I asked here to hopefully clear things up - https://phabricator.wikimedia.org/T342929#9147485.

There's also an unintentional WKData bug that slipped in, I'll put up a fix (https://github.com/wikimedia/wikipedia-ios-data/pull/10) but hold it until I can confirm that this is correct.